### DISCUSSION TITLE
Support CLI version of CLINE for System Admin tasks on a server without need for VSCode

### DISCUSSION THREAD

**Discussion by scottm-dli** [cline/cline#636] [Feature Requests]
As a user of CLINE who also has to code + deploy, would like to see a CLI version of CLINE, similar to Claude Engineer to use on a server for automation tasks.

**Comment by wardnath** [comment]
+1 would be a wonderful feature - there are already many vscode-native assistants but fewer effective cli based ones

**Reply by skuta** [reply]
Also, non .NET version!

**Comment by mtnpower** [comment]
hello i wanted to install cline on my vs code but i didnt have account so i clicked sign up and at the place when i wanted to add phone number it sent me verification code 10 minutes later and i thought that something was wrong with me so i tried few times but after that it gave me the error "Too many phone numbers entered, please contact your admin." or "Too many challenges sent for this phone number, please contact your admin." would be glad if you help me with your support
